src/H5Fquery.c

```diff
@@ -65,16 +65,16 @@
 /*******************/
 
 /*-------------------------------------------------------------------------
- * Function:	H5F_get_intent
+ * Function:  H5F_get_intent
  *
- * Purpose:	Quick and dirty routine to retrieve the file's 'intent' flags
- *          (Mainly added to stop non-file routines from poking about in the
- *          H5F_t data structure)
+ * Purpose:   Quick and dirty routine to retrieve the file's 'intent' flags
+ *            (Mainly added to stop non-file routines from poking about in the
+ *            H5F_t data structure)
  *
- * Return:	'intent' on success/abort on failure (shouldn't fail)
+ * Return:    'intent' on success/abort on failure (shouldn't fail)
  *
- * Programmer:	Quincey Koziol <koziol@ncsa.uiuc.edu>
- *		September 29, 2000
+ * Programmer:    Quincey Koziol <koziol@ncsa.uiuc.edu>
```

Comment:
Should email be removed?

Reply:
Yes

Reply:
The 1.8 branch didn't get cleaned like 1.10/1.12/develop did, though (1.10 might not be complete yet).

Reply:
Well, yes, but I think it should be done in a coordinated fashion. I.E. I have a 1.12 test PR coming, that will fix the windows error but removes the email and fixes whitespace in comments just for the test dir. This made it real clear as to the actual files that had actual changes.

Reply:
Right. I wouldn't lose sleep over the email addresses in this branch.

Reply:
Hopefully by the end of the week we can have all the branches aligned, because it makes comparison between branches so easy now.